test/unit/serialize.js

```diff
@@ -77,6 +77,9 @@ test("jQuery.param()", function() {
 	params = {"test": {"length": 3, "foo": "bar"} };
 	equal( jQuery.param( params, false ), "test%5Blength%5D=3&test%5Bfoo%5D=bar", "Sub-object with a length property" );
 
+  params = {"test": [1,2,null]};
```

Comment:
Please use spaces between elements, as well as after `{` & `[`, before `}` & `]` etc. Yes, the surrounding style is wrong, we'll fix it but it doesn't seem worth to make it even worse. ;) For more, read https://contribute.jquery.org/style-guide/js/#spacing.